components/permissions/sources.gni

```diff
@@ -1,4 +1,6 @@
 brave_components_permissions_sources = [
+  "//brave/components/permissions/permission_expiration_key.cc",
```

Comment:
are there no new deps added in these files?

Reply:
this was added https://github.com/brave/brave-core/blob/43f2ed6453ea4162ed6842074c6de8403fc220b9/components/permissions/DEPS#L6